posthog/celery.py

```diff
@@ -428,15 +429,31 @@ def check_cached_items():
     update_cached_items()
 
 
-@app.task(ignore_result=False)
-def update_cache_item_task(key: str, cache_type, payload: dict) -> None:
+@app.task(ignore_result=True)
+def update_cache_consumer_rate_limit(rate_limit: Optional[Union[str, int]]) -> None:
+    from structlog import get_logger
+
+    logger = get_logger(__name__)
+
+    if rate_limit is not None:
+        logger.info("set_update_cache_consumer_rate_limit", rate_limit=rate_limit)
+        app.control.rate_limit("posthog.celery.update_cache_item_task", rate_limit)
+
+
+@app.task(ignore_result=False, rate_limit="6/m")
```

Comment:
The default is `6/m` - is that not too slow? Or am I reading the value wrong?

Reply:
No, that's right... I took a guess at "not so fast that it'd be a problem" but init'g on start is way nicer